packages/core/src/components/Dropdown/__stories__/dropdown.stories.js

```diff
@@ -272,6 +272,37 @@ export const MultiChoiceWithDifferentStates = {
   play: multiInteractionTests
 };
 
+export const AsyncOptions = {
+  render: () => {
+    const fetchUserOptions = async () => {
+      try {
+        const response = await fetch("https://jsonplaceholder.typicode.com/users");
```

Comment:
Chromatic don't recommend dynamic apis for stories as they change, can we make it static?